packages/app/src/elements/audit.ts

```diff
@@ -91,46 +69,45 @@ export class Audit extends AuditMixin(StateMixin(Routing(View))) {
         `;
     }
 
-    async audit() {
-        const { vaults } = this.state;
-
-        const { reusedPasswords, weakPasswords, compromisedPasswords } = await this.auditVaults(vaults);
-
-        // This makes the UI update
-        this._reusedPasswords = reusedPasswords;
-        this._weakPasswords = weakPasswords;
-        this._compromisedPasswords = compromisedPasswords;
-
-        this._countElements.forEach((countElement) => {
-            animateElement(countElement, { animation: "bounce" });
-        });
```

Comment:
See my comment above for why I removed this. You can add the animation back in if you want - the best place for it that would then be in `handleRoute`, though.